### DISCUSSION TITLE
Aggregation on the relation model

### DISCUSSION THREAD

**Discussion by Ahmetadm** [prisma/prisma#16863] [Q&A]
Hi everyone, I have a model like below

```prisma
model User {
  id           Int    @id @default(autoincrement())
  first_name   String
  last_name    String
  spent_amount Float
  Sale         Sale[]
}

model Sale {
  id      Int   @id @default(autoincrement())
  total   Float
  tax     Float
  user_id Int?
  User    User? @relation(fields: [user_id], references: [ID])
}
```

is there any option to aggragate on relation model  in order to get _sum for User.spent_amount and User.Sale.total ? 
example
```
const users = await prisma.user.findMany({
  include: {
    _sum {
      select: {
        Sale: true,
      },
    },
  },
})
```

**Comment by AaravShah042** [comment]
Is there any option to do aggregate on relation int fields?

**Comment by altinthaci** [comment]
I am also having an issue like the above where I need to aggregate on relation int fields

**Comment by Sajdini** [comment]
I am also having this issue

**Comment by dan1el0123** [comment]
I also have an issue like this

**Comment by hristijanristeski** [comment]
I am also having an issue like this one.

**Comment by ludralph** [comment]
Hi @Ahmetadm 👋 

There is no way within the ORM at the moment to run this exact kind of query. This is one of those scenarios where you may need to drop into the `$queryRaw` function. However, there exists an open [issue](https://github.com/prisma/prisma/issues/15131) to support this functionality. Please add a :+1: so our engineers can prioritise it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
